WTWSMS/history/characters/old_frankish.txt

```diff
@@ -4032,7 +4032,7 @@
 		death = yes # Death date unknown
 	}
 }
-13100100 = {
+13100181 = {
```

Comment:
Why is this getting modified here? Am I missing something?

Reply:
If you check the deleted ID in discord you'll see that we talked about it. It was an error about a duplicated ID. I replaced it.

Reply:
Ok